api/users/serializers.py

```diff
@@ -509,6 +509,8 @@ def create(self, validated_data):
                 send_confirm_email(user, email=address)
         except ValidationError as e:
             raise exceptions.ValidationError(e.args[0])
+        except BlacklistedEmailError as e:
+            raise exceptions.ValidationError('This email address is blacklisted.')
```

Comment:
Throwing custom "This email address is blacklisted" error instead of reraising the default "Invalid Email" error from BlacklistedEmailError to give the user more info about what went wrong.